rpmconf/Install/zmsetup.pl

```diff
@@ -134,6 +134,12 @@
 my ($prevVersionMinor,$prevVersionMajor,$prevVersionMicro,$prevVersionBuild);
 my ($curVersionMinor,$curVersionMajor,$curVersionMicro,$curVersionMicroMicro,$curVersionType,$curVersionBuild);
 our $newinstall = 1;
+my $ldapSchemaVersion = do {
+    local $/ = undef;
+    open my $fh, "<", "/opt/zimbra/conf/attrs/zimbra-attrs-schema"
```

Comment:
Guess this will need updating for the file living in conf/

Reply:
yes it will.